changebot/github/tests/test_github_api.py

```diff
@@ -27,8 +27,26 @@ def test_get_issues(self, mock_get):
         assert self.repo.get_issues('open', 'Close?',
                                     exclude_pr=False) == [42, 55]
 
+    def test_urls(self):
+        assert self.repo._url_contents == 'https://api.github.com/repos/fakerepo/doesnotexist/contents/'
+        assert self.repo._url_pull_requests == 'https://api.github.com/repos/fakerepo/doesnotexist/pulls'
+        assert self.repo._headers is None
+
+
+# NOTE: Might hit API limit?
+class TestRealRepoHandler:
+    def setup_class(self):
+        # TODO: Use astropy/astropy-bot when #42 is merged.
+        self.repo = RepoHandler('pllim/astropy-bot', branch='changelog-onoff')
```

Comment:
Not sure how to properly test the YAML stuff except to use a real repo, which counts towards the API hit limit.

Reply:
I think this is ok though as we don't actually run the tests very often

Reply:
Travis CI for this PR seems to hit that limit a lot, but I marked it as `xfail` if that happens.